pkgs/development/python-modules/labgrid/default.nix

```
@@ -32,13 +33,9 @@ buildPythonPackage rec {
     sha256 = "sha256-yhlBqqCLOt6liw4iv8itG6E4QfIa7cW76QJqefUM5dw=";
   };
 
-  patches = [
-    # Pyserial within Nixpkgs already includes the necessary fix, remove the
-    # pyserial version check from labgrid.
-    ./0001-serialdriver-remove-pyserial-version-check.patch
-  ];
+  nativeBuildInputs = [ setuptools-scm pythonRelaxDepsHook ];
 
-  nativeBuildInputs = [ setuptools-scm ];
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
Could you please use `pyproject = true;` instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters